Resources/Private/Javascript/Components/CartSummary.js

```diff
@@ -24,34 +24,34 @@ const CartSummary = () => {
   },[])
 
   return (
-    <div>
+    <div className='c-cart-summary'>
       <Overlay loading={loading} />
 
-      {Object.keys(cartData).length > 1 ?
-        <dl>
-          <dt>Different products:</dt>
-          <dd>{cartData.lineItems.length}</dd>
+      {Object.keys(cartData).length > 0 ?
+        <dl className='c-cart-summary__list'>
+          <dt className='c-cart-summary__label c-cart-summary__label--count'>Different products:</dt>
```

Comment:
can we put a little more structure in there, like warp the term and description in a container, so that i can style them as a block?

Reply:
Also we need to find a solution for the labels so that they are at least translatable